pipenv/utils.py

```diff
@@ -946,3 +947,26 @@ def download_file(url, filename):
 
     with open(filename, 'wb') as f:
         f.write(r.content)
+
+
+def need_update_check():
+    mkdir_p(PIPENV_CACHE_DIR)
+    p = os.sep.join((PIPENV_CACHE_DIR, '.pipenv_update_check'))
+    if not os.path.exists(p):
+        return True
+    out_of_date_time = time() - (24 * 60 * 60)
+    if os.path.isfile(p) and os.path.getmtime(p) <= out_of_date_time:
+        return True
+    else:
+        return False
+
+
+def checked_for_updates():
```

Comment:
Only note I have is maybe renaming this. The name implies it's Boolean in nature, but it will always be true. We probably don't have a reason to return True either at the bottom.

Reply:
@nateprewitt renamed to `touch_update_stamp()`